persistence/src/main/java/cz/muni/fi/pa165/mushrooms/entity/MushroomHunter.java

```diff
@@ -10,15 +10,15 @@
 public class MushroomHunter {
 
     @Id
-    @GeneratedValue(strategy= GenerationType.IDENTITY)
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
 
     @NotNull
-    @Column(nullable=false,unique=true)
+    @Column(nullable = false, unique = true)
     private String firstName;
 
     @NotNull
-    @Column(nullable=false,unique=true)
+    @Column(nullable = false, unique = true)
     private String surname;
 
     private boolean isAdmin;

```

Comment:
This file should not be edited in this commit, wut whatever. Names should be not unique and not nullable.